src/absil/il.fs

```diff
@@ -18,6 +18,7 @@ open System.Collections
  
 let logging = false 
 
+#if FX_RUNNING_ON_MONO
```

Comment:
My goal ... such as it was, was to isolate the code that is required for running on Mono.  It is my hope that Windows Desktop and Mono will do a good job of running coreclr and that coreclr will replace those builds eventually.

I can certainly rename RUNNING_ON_MONO if that reduces confusion.  Updating the desktop build defines is probably not that big of a deal.

I would prefer a positive ifdef perhaps: ENABLE_MONO_SUPPORT rather than a negative one because the lkg compiler only supports the old style preprocessor.